pypsa/linopf.py

```diff
@@ -1028,6 +1028,8 @@ def save_optimal_capacities(n, iteration, status):
                                 f'and termination {termination_condition}')
         if track_iterations:
             save_optimal_capacities(n, iteration, status)
+            n.global_constraints = n.global_constraints.rename(columns={'mu': f'mu_{iteration}'})
+        final_duals = n.global_constraints.mu.copy() #save second last iteration of duals (needed for lv_limit duals)
```

Comment:
When you track iterations, don't you get rid of the 'mu' column in the first row and in the second you try to access it? Does that work? I haven't tried.

Reply:
Huh, true, but it worked when I tried. That's strange. I'll change it, just to be sure